tests/system/feature_store/test_feature_store.py

```diff
@@ -104,9 +104,27 @@ def _ingest_quotes_featureset(self):
             FeaturesetValidator()
         )
 
-        quotes_set.add_aggregation("asks1", "ask", ["sum", "max"], "1h", "10m")
-        quotes_set.add_aggregation("asks2", "ask", ["sum", "max"], "5h", "10m")
-        quotes_set.add_aggregation("bids", "bid", ["min", "max"], "1h", "10m")
+        quotes_set.add_aggregation("ask", ["sum", "max"], "1h", "10m", name="asks1")
+        quotes_set.add_aggregation("ask", ["sum", "max"], "5h", "10m", name="asks2")
+        quotes_set.add_aggregation("bid", ["min"], "1h", "10m")
+
+        try:
+            # no name parameter, different window
+            quotes_set.add_aggregation("bid", ["max"], "5h", "10m")
+            assert False
+        except mlrun.errors.MLRunInvalidArgumentError:
+            pass
+
+        try:
+            # no name parameter, different period
+            quotes_set.add_aggregation("bid", ["max"], "1h", "5m")
+            assert False
+        except mlrun.errors.MLRunInvalidArgumentError:
+            pass
```

Comment:
```suggestion
        with pytest.raises(mlrun.errors.MLRunInvalidArgumentError):
            # no name parameter, different period
            quotes_set.add_aggregation("bid", ["max"], "1h", "5m")
```

This is more correct - it ensures we're getting the right exception.